packages/babel-plugin-proposal-object-rest-spread/src/index.js

```diff
@@ -180,6 +180,19 @@ export default declare((api, opts) => {
       keyExpression = t.arrayExpression(keys);
     }
 
+    // Hoist definition of excluded keys, so that it's not created each time.
+    if (!t.isProgram(path.scope.block)) {
+      const id = path.scope.generateUidIdentifier();
```

Comment:
Nit: Give it a naming hint:
```suggestion
      const id = path.scope.generateUidIdentifier("exclude");
```